packages/ubuntu_wizard/pubspec.yaml

```diff
@@ -33,6 +33,7 @@ dependencies:
       path: packages/ubuntu_logger
   wizard_router: ^0.6.0
   yaru: ^0.2.0
+  yaru_icons: ^0.0.8
```

Comment:
This could be a opportunity to upgrade to the new 0.1.2 yaru_icons where the path of the widgets changed :)

Reply:
Ah yeah, thanks for the reminder! :) https://github.com/canonical/ubuntu-desktop-installer/pull/594